src/Pages/jiminkim/Main/Feeds/Comments/CommentList.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+
+export default class CommentsList extends React.Component {
+  render() {
+    const { feedId, commentsList, deleteComment, handleCommentHeart } =
+      this.props;
+
+    return (
+      <>
+        <div className="feeds-comment-list">
+          <ul className="list-style-none">
+            {commentsList?.map((comment, no) => (
+              <li key={no}>
+                <span className="nickname bold">{comment.nickname}</span>
+                <span className="comment">{comment.comment}</span>
+                <div className="button-list">
+                  <button
+                    className="comment-delete-button"
+                    id={comment.no}
+                    onClick={e => deleteComment(e)}
```

Comment:
```suggestion
                    onClick={deleteComment}
```
함수 자체를 넘겨줘도 동일한 동작을 할 것 같습니다

Reply:
감사합니다! 해당 로직은 고민 후에 deleteComment 인자값으로 comment.no을 넘기도록 변경하였습니다. 
`onClick={() => deleteComment(comment.no)}`